laterpay/application/Controller/Install.php

```diff
@@ -463,10 +496,10 @@ public function install() {
         add_option( 'laterpay_landing_page',                            '' );
 
         // advanced settings
-        add_option( 'laterpay_api_sandbox_url',                         'https://api.sandbox.laterpaytest.net' );
-        add_option( 'laterpay_api_sandbox_web_url',                     'https://web.sandbox.laterpaytest.net' );
-        add_option( 'laterpay_api_live_url',                            'https://api.laterpay.net' );
-        add_option( 'laterpay_api_live_web_url',                        'https://web.laterpay.net' );
+        add_option( 'laterpay_api_sandbox_backend_url',                 'https://api.sandbox.laterpaytest.net' );
```

Comment:
This should really be "laterpay_sandbox_backend_api_url".
Please change all the variables accordingly. The scheme is
**[prefix: laterpay]_[environment: sandbox or live]_[API name: dialog_api or backend_api]_[suffix: url]**